ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -65,6 +65,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+  timeout: 8h0m0s
```

Comment:
We should remove the limit here.

Reply:
we change default value to 23h, so add 8h for non-desctrucitve test and long duration test.
please refer to https://github.com/openshift/release/pull/31934#issuecomment-1236007314 and https://github.com/openshift/release/pull/31934#issuecomment-1236333913

Reply:
`openshift-e2e-test-qe` runs multiple tests, ginkgo (in openshift-tests-private) is only part of it. 
Set ginkgo to timeout 23 hour, while set the total to timeout 8 hour, it does not looks a great plan IMO.

Reply:
it does not change ginkgo step to 23h. it is for test level. I change to 23h in the code https://github.com/openshift/release/pull/31934/files#diff-3fe4522c3fa00ecd918a986bc31b4cde479c02bcace2b0497bd136760edbd71f

Reply:
we use one step to cover destructive, longduration, and other cases. although the max timetout is 23h for the step, it does not run so long time for non-destructive, and non-longduration, so it is OK to add 8h for non-destructive, and non-longduration test.

Reply:
so, you suggest we make different step for different case, and set different timeout for step, so we do not need specific timeout for test?